server/controllers/cohort.js

```diff
@@ -159,17 +159,22 @@ const getCohortDetails = (req, resp) => {
       if (isArchived) {
         return resp.status(200).json({ _id, isArchived, name });
       }
-
+      const arrayOfMemberIds = membersCollection.map(member => member._id);
       const isOwner = checkIfArrayContainsUserId(ownerIds, req.user._id);
+      const isMember = checkIfArrayContainsUserId(
```

Comment:
this check in unnecessary, if user is in cohort (the cohort was find via query), them is a member for sure. So isMember should be set to true for each cohort member(user).